ansible/scripts/build-and-run.sh

```diff
@@ -5,6 +5,12 @@ PROFILE=${PROFILE:-small}
 BENCHNAME=${BENCHNAME:-$(date +"%Y-%m-%dT%H:%M:%S")}
 export BENCHNAME
 
+# dropping the tpcc schema before test run. and ignore the error.
+if [ "$DROP_TPCC_SCHEMA_FOR_EACH_BENCHMARK" = "true" ]; then
+  echo "Before starting the benchmark...deleting the oracle user tpcc."
+  ./drop_tpcc_user.sh || echo "Ignoring failure in drop_tpcc_user.sh"
+fi
+
```

Comment:
Just curious what was the reasoning of moving this code from `build.sh`? initial reason to add this to `build.sh` is that we might need this when we do the build time of schema.

Reply:
There is no confirmation before deletion and user might accidentally run the `build.sh` delete the schema without knowing it. similarly I have done :). so just giving more protection putting into better place `build-and-run.sh` where user intension is more clear to build the schema and run it.